portfolio/src/main/webapp/style.css

```diff
@@ -76,4 +84,20 @@
 
 .sticky + .content {
   padding-top: 5rem;
+}
+
+#recommender-system {
+  background-color: peachpuff;
+}
+
+#swe-dev-portal {
+  background-color: oldlace;
+}
+
+#five-minutes-a-day {
```

Comment:
Not in alphabetic order?

Reply:
Within each of the blocks, I believe the declarations are in order. Do the blocks themselves also need to be in order?

Reply:
No, the blocks doesn't have to be in order.

Reply:
Ack.